internal/install/install.go

```diff
@@ -92,11 +94,11 @@ func ServiceLogsDir() (string, error) {
 
 // TestReportsDir returns the location of the directory to store test reports.
 func TestReportsDir() (string, error) {
```

Comment:
I wonder if `TestReportsDir()` should be present in `install` which is related rather to `~/.elastic-package`. The `test-reports` directory won't be in that place anymore.

Reply:
Fair point, I'll move it out of that package.

Reply:
Done in 2a4b779.